vllm/entrypoints/cli/serve.py

```diff
@@ -134,3 +162,143 @@ def signal_handler(signum, frame):
     finally:
         logger.info("Shutting down.")
         engine_manager.close()
+
+
+def run_multi_api_server(args: argparse.Namespace):
+
+    assert not args.headless
+    num_api_servers = args.api_server_count
+    #assert num_api_servers > 1
```

Comment:
nit: cruft

Will `num_api_servers == 0` be equivalent to headless?

Reply:
> Will `num_api_servers == 0` be equivalent to headless?

Yes, I made that an alternative way of specifying headless [here](https://github.com/vllm-project/vllm/pull/17546/files#diff-9786c2997a0f0d2b41beb8185f67b6df9cfd87601b270ffcd62b7515e959c672R54), but  that won't take this path.

I'll change the assert here to > 0. Although I'm not using this path for the == 1 case, it does still work and for consistency we could consider later changing so that the "launcher" approach is used in all cases (once V0 is gone).